src/template/ProjectArticle.tsx

```diff
@@ -109,17 +110,23 @@ const ProjectArticle = ({ data }: PageProps<Queries.SingleProjectQuery>) => {
         title={bannerTitle}
         sx={{ backgroundColor: "purple.A100", mt: { xs: "48px", lg: "96px" } }}
       >
-        <Button
-          size='large'
-          sx={{
-            borderRadius: "100px",
-            background: "#8769FE",
-            color: "#fff",
-            fontSize: "18px",
-          }}
+        <SeoLink
+          isExternal={true}
+          link={queryData?.corsi?.[0]?.couponLink}
```

Comment:
@leonardodemarco00 
Sono abbastanza indeciso. Essendoci già il componente fixed che cerca di buttare fuori l'utente, secondo te anche questo banner dovrebbe buttare l'utente fuori ? 
Sembra che ci manchi una CTA che porti l'utente sulla landing del corso dio cane

Reply:
@omardeangelis bella questione
Effettivamente avendo già l'altro elemento fixed, potremmo fare in modo che il banner al fondo riporti alla pagina del corso. Tanto anche li c'è il pulsante molesto per far riscattare il coupon.

Reply:
Sono d'accordo, facciamo così, mi sembra strano che l'utente non posso andare sulla landing verticale del corso da dentro l'articolo. Forse quel banner non è la soluzione più chiara possibile; in un prossimo layer dovremmo studiarci qualcosa

Reply:
@omardeangelis  No magari non è la soluzione migliore ma per ora direi che ci sta